components/camera/collectors.go

```diff
@@ -28,6 +30,8 @@ func (m method) String() string {
 		return "NextPointCloud"
 	case readImage:
 		return "ReadImage"
+	case readImages:
```

Comment:
We should likely rename ReadImage->GetImage in a separate PR (since that has UI and customer implications), but should name this one `getImages` and `GetImages` to match the API.